src/libstore/store-api.cc

```diff
@@ -86,18 +86,25 @@ string storePathToHash(const Path & path)
 void checkStoreName(const string & name)
 {
     string validChars = "+-._?=";
+
+    auto baseError = format("The path name '%2%' is invalid: %3%. "
```

Comment:
Please use `fmt("<format-string>", args...)` in new code.

Reply:
Right now I depend on the final substitution being "curried", is there a way to support that, or should I not do the partial application?

Reply:
Thanks, didn't notice that.